config/locales/en/steps.yml

```diff
@@ -10,3 +10,7 @@ en:
           page_title: Enter your client’s details
           heading: Enter your client’s details
           full_name_legend: Full name
+      has_nino:
+        edit:
+          page_title: Enter your client’s NINO
+          has_nino: Do you have your client's National Insurance number? 
```

Comment:
As we are using the legend as the page heading, this `has_nino` locale in reality is not being used so can be removed.